tools/topology/topology2/cavs/cavs-nocodec.conf

```diff
@@ -27,6 +27,10 @@
 <dai.conf>
 <host.conf>
 
+Define {
+	MCLK 38400000
+}
+
```

Comment:
Could that define be a property of the platform and included, rather than added in CMakeLists?
We've got to reduce the number of options in CMakeLists, this should be used for board variants and not for silicon variants IMHO.

@aiChaoSONG @ranj063 @lgirdwood FYI

Reply:
yep, that's good for me. How would we deal with nocodec platforms if we want to try multiple MCLKS (for testing completeness). I suppose nocodec is a special case here as its expected to be broad in terms of configuration options.

Reply:
for this topology, we use 'default' and 'representative' settings for the SSP for each platform. The goal is not to test all possible permutations of the MCLK ID, values, and then same for bclk settings. It's hard enough to support mixer and loopback with this topology. 

If there is a desire to proceed with more SSP validation, it will have to be in the 'development' directory and have an owner that keeps track of all variations.